src/global/global_init.cc

```diff
@@ -215,6 +227,20 @@ global_init(const std::map<std::string,std::string> *defaults,
   if (g_conf()->fatal_signal_handlers) {
     install_standard_sighandlers();
   }
+
+#ifdef HAVE_BREAKPAD
+  if (g_conf()->breakpad) {
+    google_breakpad::MinidumpDescriptor descriptor(g_conf()->crash_dir);
+    g_ceph_context->_ex_handler.reset(
```

Comment:
This is fine, but also consider:
_ex_handler = std::make_unique<google_breakpad::ExceptionHandler(...your args...);